docs/functions/function-crd.md

```diff
@@ -90,6 +90,22 @@ Function Mesh provides the following fields for Stateful functions in the CRD de
 | `statefulConfig.pulsar.javaProvider.className` | The Java class name of the state storage provider implementation. The class must implement the `org.apache.pulsar.functions.instance.state.StateStoreProvider` interface. If not, `org.apache.pulsar.functions.instance.state.BKStateStoreProviderImpl` will be used. |
 | `statefulConfig.pulsar.javaProvider.config` | The configurations that are passed to the state storage provider. |
 
+## Window function configurations
+
+Function Mesh provides the following fields for window functions in the CRD definition.
+
+| Field                           | Description                                                                                                                                                                 |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `actualWindowFunctionClassName` | Required. The runner class name of the implemented window function.                                                                                                         |
```

Comment:
This field can be optional and the default value is the same as the `spec.className `